### PR TITLE
Update getLocals usage

### PR DESCRIPTION
**Pull request by gabro**

### CODE REVIEW

File: frontend/tutorial/step_7.md [hunk 1]
```diff
@@ -14,17 +14,19 @@ const baseProps = {
 
 describe('Hello', () => {
 
+  const getLocals = Hello.prototype.getLocals;
```
Comment (giogonzo) [inline frontend/tutorial/step_7.md]: this adds some burden IMO,
also it is potentially error-prone: in many `getLocals` around the apps we access `this`, typically for other helper methods

It looks safer if we just suggest to `instance.getLocals(props)`?

Comment (gabro) [reply frontend/tutorial/step_7.md]: right, I forgot that we often reference `this` in `getLocals` for other things that are not `props`. I'll change it.

Comment (gabro) [reply frontend/tutorial/step_7.md]: @giogonzo done, check it out now

**Review by giogonzo — APPROVED** [review]
Looks good to me 👍

**Comment by giogonzo** [comment]
@gabro "full diff" links will be (even more, they are already) outdated after this changes, but this is another story I guess..

**Comment by gabro** [comment]
> @gabro "full diff" links will be (even more, they are already) outdated after this changes, but this is another story I guess..

yes, I think we need to fix them all (out of scope I'd say)

**Pull request merged**